StoreApp.BLL/Services/CategoryService.cs

```diff
@@ -0,0 +1,57 @@
+﻿using StoreApp.BLL.DTO;
+using StoreApp.BLL.Interfaces;
+using System.Collections.Generic;
+using StoreApp.DAL.Intefaces;
+using StoreApp.DAL.Entities;
+using AutoMapper;
+
+namespace StoreApp.BLL.Services
+{
+    public class CategoryService : ICategoryService
+    {
+        IUnitOfWork DataBase { get; set; }
+        public CategoryService(IUnitOfWork uow)
+        {
+            DataBase = uow;
+        }
```

Comment:
The methods must be splitted by using indents

Reply:
fixed